Public/Src/Cache/ContentStore/Library/Stores/QuotaManagement/LegacyQuotaKeeper.cs

```diff
@@ -192,10 +192,11 @@ public override async Task<ReserveTransaction> ReserveAsync(long contentSize)
                 throw new CacheException($"Failed to reserve space for content size=[{contentSize}], result=[{result}]");
             }
 
-            return new ReserveTransaction(contentSize, Evicted);
+            return new ReserveTransaction(contentSize, OnContentEvicted);
         }
 
-        private void Evicted(long size)
+        /// <inheritdoc />
+        public override void OnContentEvicted(long size)

```

Comment:
And here, pure name changing unrelated to feature adds an entire file to PR. Am I missing how this is related?

Reply:
This is not "unrelated". I've made the method public and virtual because FileSystemContentSTore now can notify the keeper that the size has changed.

---
In reply to: [283071106](https://github.com/microsoft/BuildXL/pull/289#discussion_r283071106) [](ancestors = 283071106)